<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveMainWindow</class>
 <widget class="QMainWindow" name="CurveMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1181</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="buttonWidget" native="true">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>0</y>
       <width>200</width>
       <height>751</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="drawBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>绘制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clcBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>清屏</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cutBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>截屏</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>曲线次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le1"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>曲线类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb1">
         <item>
          <property name="text">
           <string>B样条曲线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>贝齐尔曲线</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>202</width>
        <height>40</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>B样条曲线类型：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb2">
         <item>
          <property name="text">
           <string>准均匀B样条曲线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>均匀B样条曲线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Riensenfeld参数化非均匀B样条</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hartley-Judd参数化非均匀B样条</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>分段贝齐尔曲线</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>710</y>
        <width>158</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>返回</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>请填写大于等于0的整数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="curveWidget" native="true">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>1011</width>
       <height>751</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>751</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="upWidget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="downWidget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>曲线绘制</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
